### ISSUE TITLE
Path file case detection

### ISSUE THREAD

**Issue by JoinedSenses** [alliedmodders/sourcemod#831]
# Description
SM 1.9.0.6237
Issue with handling of path files - not sure scope of bug, however it was noticed during sm_rcon sm plugins unload PluginFolder/plugin.smx

# Problematic Code (or Steps to Reproduce)
Folder exists in /plugins dir with capital letters. Inside is plugin. (eq plugins/PluginFolder/plugin.smx)
run sm_rcon sm plugins unload PluginFolder/plugin
error that plugin is not loaded
run sm_rcon sm plugins unload pluginfolder/plugin
success

# Example:
```
sm_rcon sm plugins unload SourceIRC/sourceirc-relayall
[SM] Plugin SourceIRC\sourceirc-relayall.smx is not loaded.
sm_rcon sm plugins unload sourceirc/sourceirc-relayall
[SM] Plugin SourceIRC -> Relay All will be unloaded on the next frame.
```

**Comment by Headline** [comment]
Is your server running Linux? The Linux OS differentiates between `/thisfolder/` and `/THISFOLDER/`, however on windows they should resolve to the same path.

There was a commit recently about this, and another one on the way. If you're running linux, however, this is not a bug.

**Comment by JoinedSenses** [comment]
Im on windows

**Comment by Headline** [comment]
I'm sorry, I have misread your issue completely.

What you've described is fixed in #740
